src/main/java/fi/otavanopisto/kuntaapi/server/integrations/management/ManagementMenuProvider.java

```diff
@@ -111,99 +108,97 @@ public MenuItem findOrganizationMenuItem(OrganizationId organizationId, MenuId m
     ApiResponse<List<fi.otavanopisto.mwp.client.model.Menuitem>> response =  
-        mwpApi.getApi(organizationId).kuntaApiMenusMenuIdItemsGet(mwpMenuId.getId());
+        managementApi.getApi(organizationId).kuntaApiMenusMenuIdItemsGet(managementMenuId.getId());
     
     if (!response.isOk()) {
       logger.severe(String.format("Menu finding failed on [%d] %s", response.getStatus(), response.getMessage()));
     } else {
-      for (fi.otavanopisto.mwp.client.model.Menuitem mwpItem : response.getResponse()) {
-        if (mwpItemId.getId().equals(String.valueOf(mwpItem.getId()))) {
-          return translateMenuItem(mwpItem);
+      for (fi.otavanopisto.mwp.client.model.Menuitem managementMenuItem : response.getResponse()) {
+        if (managementItemId.getId().equals(String.valueOf(managementMenuItem.getId()))) {
+          return translateMenuItem(managementMenuItem);
         }
       }
     }
     
     return null;
   }
   
-  private List<Menu> translateMenus(List<fi.otavanopisto.mwp.client.model.Menu> mwpMenus) {
+  private List<Menu> translateMenus(List<fi.otavanopisto.mwp.client.model.Menu> managementMenus) {
     List<Menu> result = new ArrayList<>();
     
-    for (fi.otavanopisto.mwp.client.model.Menu mwpMenu : mwpMenus) {
-      result.add(translateMenu(mwpMenu));
+    for (fi.otavanopisto.mwp.client.model.Menu managementMenu : managementMenus) {
+      result.add(translateMenu(managementMenu));
     }
     
     return result;
   }
 
-  private Menu translateMenu(fi.otavanopisto.mwp.client.model.Menu mwpMenu) {
+  private Menu translateMenu(fi.otavanopisto.mwp.client.model.Menu managementMenu) {
     Menu menu = new Menu();
     
-    MenuId mwpId = new MenuId(MwpConsts.IDENTIFIER_NAME, String.valueOf(mwpMenu.getId()));
-    MenuId kuntaApiId = idController.translateMenuId(mwpId, KuntaApiConsts.IDENTIFIER_NAME);
-    if (kuntaApiId == null) {
-      logger.info(String.format("Found new menu %d", mwpMenu.getId()));
-      Identifier newIdentifier = identifierController.createIdentifier(mwpId);
-      kuntaApiId = new MenuId(KuntaApiConsts.IDENTIFIER_NAME, newIdentifier.getKuntaApiId());
+    MenuId managementMenuId = new MenuId(MwpConsts.IDENTIFIER_NAME, String.valueOf(managementMenu.getId()));
+    MenuId kuntaApiMenuId = idController.translateMenuId(managementMenuId, KuntaApiConsts.IDENTIFIER_NAME);
+    if (kuntaApiMenuId == null) {
+      logger.info(String.format("Could not translate management menu %d into kunta api id", managementMenu.getId()));
+      return null;
     }
     
-    menu.setId(kuntaApiId.getId());
-    menu.setSlug(mwpMenu.getSlug());
+    menu.setId(kuntaApiMenuId.getId());
+    menu.setSlug(managementMenu.getSlug());
     
     return menu;
   }
   
-  private List<MenuItem> translateMenuItems(List<fi.otavanopisto.mwp.client.model.Menuitem> mwpMenuItems) {
+  private List<MenuItem> translateMenuItems(List<fi.otavanopisto.mwp.client.model.Menuitem> managementMenuItems) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused private "translateMenuItems" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)